js/maps/tiles.js

```diff
@@ -1,32 +1,125 @@
-// This file only contains lists of spells and their actions / attributes
+// This file only contains lists of tiles and their actions / attributes
 if(window.app == null ||
    window.app == undefined) {
   window.app = {};
 }
 
 var TILES = {
+// Basic Environment Tiles
   DIRT: {
```

Comment:
I'm super stoked that you filled in some of these tiles. Also the maps above. But we should alphabetize these attributes....  good sublime practice!! using `ctrl+shift+↑` or `ctrl+shit+↓` you can move lines up or down. To look something like:

    DIRT: {
      action: null,
      imageUrl: "", 
      info: "Just some dirt",
      name: "DIRT",
      solid: false
    }

OR

    DIRT: {
      // variables:
      imageUrl: "", 
      info: "Just some dirt",
      name: "DIRT",
      solid: false,
      // functions:
      action: null,
    } 
(don't need the comments)